samples/bluetooth/peripheral_mds/README.rst

```diff
@@ -0,0 +1,187 @@
+   On when the development kit is connected.
+
+Button 1:
+   Press this button to start time measuring.
+   The second press stops time measuring.
+
+   During the pairing procedure, press this button to accept pairing.
+
+Button 2:
+   Triggers the ``Button2StateChanged`` trace event.
+
+   During the pairing procedure, press this button to reject pairing.
+
+Button 3:
+   Every press of this button is counted under the ``Button3PressCount`` metric.
+
+Button 4:
+   Simulate a development kit crash by triggering a hardfault exception by division by zero.
+
+
+Configuration
+*************
+
+|config|
+
+The Memfault SDK allows configuring some of its options using Kconfig.
+For the options not configurable using Kconfig, use the :file:`samples/bluetooth/peripheral_mds/memfault_config/memfault_platform_config.h` file.
+See `Memfault SDK`_ for more information.
+
+To send data to the Memfault cloud through a Bluetooth gateway, you must configure a project key using the :kconfig:option:`CONFIG_MEMFAULT_NCS_PROJECT_KEY` option.
+You can find your project key in the project settings at `Memfault Dashboards`_.
+You also need to set the following static configuration option for this sample:
+
+   * :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID` - Memfault device ID.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_mds`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+Before testing, ensure that your device is configured with the project key of your Memfault project.
+
+|test_sample|
+
+Testing with Memfault WebBluetooth Client
+-----------------------------------------
+
+1. |connect_terminal_ANSI|
+#. Reset your development kit.
+#. Observe that the sample starts.
+#. Open a recent version of the `Google Chrome browser`_.
+#. Run the `Memfault WebBluetooth Client`_  script to forward memfault diagnostic data to the cloud.
+   For more details check the script source code here: `Memfault WebBluetooth Client source code`_.
+#. Make sure that your development kit is advertising.
+#. In the browser, click the :guilabel:`Connect` button and select your device from the list.
+#. Use the buttons on your development kit to confirm or reject a pairing request:
+
+   .. code-block:: console
+
+      Pairing confirmation required for 6D:99:66:6E:19:72 (random)
+      Press Button 1 to confirm, Button 2 to reject.
+
+#. Upon connection, data already collected by the `Memfault SDK`_ is forwarded to the cloud for further the analysis.
+   When connected, the new data is periodically flushed to the cloud with the interval configured by the option :kconfig:option:`CONFIG_BT_MDS_DATA_POLL_INTERVAL`.
+#. Upload the symbol file generated from your build to your Memfault account so that the information from your application can be parsed.
+   The symbol file is located in the build folder :file:`peripheral_memfault/build/zephyr/zephyr.elf`.
+   a. In a web browser, navigate to `Memfault`_.
```

Comment:
```suggestion

   a. In a web browser, navigate to `Memfault`_.
```
I believe this missing empty line made the doc build crash.

Reply:
Fixed